src/datasources/config-api/config-api.service.ts

```diff
@@ -38,10 +39,29 @@ export class ConfigApi implements IConfigApi {
 
   async getChain(chainId: string): Promise<Chain> {
     try {
-      const key = `${chainId}_chain`;
+      const cacheKey = `${chainId}_chain`;
       const field = '';
       const url = `${this.baseUri}/api/v1/chains/${chainId}`;
-      return await this.dataSource.get(key, field, url);
+      return await this.dataSource.get(cacheKey, field, url);
+    } catch (error) {
+      throw this.httpErrorFactory.from(error);
+    }
+  }
+
+  async getSafeApps(
+    chainId: string,
+    clientUrl?: string,
+    url?: string,
+  ): Promise<SafeApp[]> {
+    try {
+      const cacheKey = `${chainId}_safe_apps`;
+      const field = `${clientUrl}_${url}`;
+      return await this.dataSource.get(
+        cacheKey,
+        field,
+        `${this.baseUri}/api/v1/safe-apps/?chainId=${chainId}`,
```

Comment:
Why not have `chainId` in the `params`?

Reply:
Changed in [32a5f1eb7e](https://github.com/5afe/safe-client-gateway-nest/commit/32a5f1eb7eb7e1e2b02d5e785aba5335a72d3cac)